src/main/java/duke/util/Saving.java

```diff
@@ -0,0 +1,13 @@
+package duke.util;
+
+/**
+ * Interface for saving data by making it into a string to be saved into a file.
+ */
+public interface Saving {
```

Comment:
Thanks for the suggestion, however, I was going for a more generic OOP approach. I wanted Storage class to be as generic as possible and be able to save and load other classes besides Task (although I know for this individual project we don't need to do that).

Thus, I created a Saving and Loading interface so the class that implements them can have their own behaviour or formatting/reading for saving and loading. I separated Saving and Loading into different interfaces as I felt it would be better too.

Thanks for the suggestion!